requirements.txt

```diff
@@ -1,2 +1,3 @@
 yapf==0.25.0
 tqdm==4.46.0
+wheel~=0.34
```

Comment:
Is this diff expected?
I think this diff is same as #55 .

Reply:
@nutti this PR here contains two commits. I rebased this branch on top of the other fix branch, so that we are having a consistent PR in the end.

I know that you do not like multiple commits in one PR, so I created PR #55 . 
* If you merge #55 and then squash+merge #47 then you will have two commits in the end.
* If you merge #47 without squashing, then you will have the identical commits in the end.

* If you squash+merge #47, then #55 will have conflicts and will not be mergeable and we only have one commit.

Reply:
Understood. Thanks for consideration!

> If you merge #55 and then squash+merge #47 then you will have two commits in the end.

I prefer this option. So, I will merge #55 at first.